protocol/pubsub/v2/go.mod

```diff
@@ -8,13 +8,14 @@ replace github.com/cloudevents/sdk-go/v2 => ../../../v2
 
 require (
 	cloud.google.com/go/pubsub v1.50.0
```

Comment:
I was expecting this to be removed too? Where are we still using it?

Reply:
ah missed a file